src/CHANGELOG.js

```diff
@@ -7,6 +7,11 @@ import SpellLink from 'common/SpellLink';
 import Contributor from 'interface/contributor/Button';
 
 export default [
+  {
+    date: new Date('2019-06-19'),
```

Comment:
Might want to set this to the release date so it's actually visible on release :)

Reply:
fair enough lol